web_client/src/store/index.ts

```diff
@@ -231,13 +231,14 @@ function queueLoadScan(scan, loadNext = 0) {
         ];
       }
     } else {
-      let offset = 1;
-      while (!nextScan || !includeScan(nextScan.id)) {
+      let newIndex = scansInSameExperiment.indexOf(scan.id);
```

Comment:
Is it guaranteed that `scan.id` will be present in `scansInSameExperiment`? Just trying to consider what would happen if `newIndex` became `-1` here, or if that's even possible.

Reply:
Hmm, i think it's guaranteed by the definition of `scansInSameExperiment`, but it couldn't hurt to protect ourselves. Changed in [5b9e1e3](https://github.com/OpenImaging/miqa/pull/627/commits/5b9e1e35f3a6889740f80b158c012c8c4a056864)